Sources/Subs-Calendar.php

```diff
@@ -980,6 +1073,10 @@ function getEventProperties($event_id)
 			c.id_event, c.id_board, c.id_topic, MONTH(c.start_date) AS month,
 			DAYOFMONTH(c.start_date) AS day, YEAR(c.start_date) AS year,
 			(TO_DAYS(c.end_date) - TO_DAYS(c.start_date)) AS span, c.id_member, c.title,
+			MONTH(c.end_date) AS end_month, DAYOFMONTH(c.end_date) AS end_day, YEAR(c.end_date) AS end_year,
+			HOUR(start_time) AS hour, MINUTE(start_time) AS minute, SECOND(start_time) AS second,
```

Comment:
two things:
1. you should use the table alias (c) in this and next line  start_time -> c.start_time ...

2. in my eyes this table call is not good use, because the heavy use of time function month, dayofmonth, year, hour, minute and second().
Better would be to keep the line 1079 and remove 1077, 1078,
the remaining information generate in php.
Reasons:
- the timefunction are different in mysql and pg 
- sql statement get cleaner